nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ExecuteStreamCommand.java

```diff
@@ -351,6 +350,16 @@ public void onTrigger(ProcessContext context, final ProcessSession session) thro
         builder.directory(dir);
         builder.redirectInput(Redirect.PIPE);
         builder.redirectOutput(Redirect.PIPE);
+        final File errorOut;
+        try {
+            errorOut = File.createTempFile("out", null);
+            errorOut.deleteOnExit();
```

Comment:
I would like to see this deleted after every iteration of the processor because it could a very long time brefore deleteOnExit gets executed.

Reply:
Good point, I will do that